Composer/packages/server/src/server.ts

```diff
@@ -57,6 +57,8 @@ export async function start(electronContext?: ElectronContext): Promise<number |
   setEnvDefault('COMPOSER_BUILTIN_EXTENSIONS_DIR', path.resolve(__dirname, '../../../../extensions'));
   // Composer/.composer/extensions
   setEnvDefault('COMPOSER_REMOTE_EXTENSIONS_DIR', path.resolve(__dirname, '../../../.composer/extensions'));
+  setEnvDefault('COMPOSER_TEMP_DIR', path.resolve(__dirname, '../../../.composer/.temp'));
```

Comment:
nit: I'd do this similar to .git structure, .composer dir and then normal dirs under (non dot directories):
.composer/backup
.composer/temp

Reply:
works for me